NineAnimator/Models/Anime Source/gogoanime/GogoAnime+Episode.swift

```diff
@@ -73,6 +73,11 @@ extension NASourceGogoAnime {
                     throw NineAnimatorError.responseError("No streaming sources found for this anime")
                 }
                 
+                //GogoAnime's "Vidstreaming" server is a collection of backup links to different servers, however there is currently no way for the user to select a backup link, so we are removing this server from the Dictionary of sources.
+                
+                //Keep in mind that GogoAnime's "Gogo server" does use the VidStreamingParser
+                streamingSources.removeAll { $0.0 == "Vidstreaming" }
```

Comment:
This will bypass the empty episode list check. The `guard !streamingSources.isEmpty` line prevents the app from crashing. Consider replacing the `streamingSources.removeAll { $0.0 == "Vidstreaming" }` and integrating this function into the `.compactMap { }` above or add an extra `.filter { }` call after `.compactMap { }`.

Reply:
I've updated the code as requested